### PR TITLE
[Fix] Add Scroll chainType and L1GasOracle 

### PR DESCRIPTION
**Pull request by stackman27**

### CODE REVIEW

**Review by amit-momin — APPROVED** [review]
Looks good to me but think we should wait for a ship pod approval before merging

File: core/chains/evm/gas/rollups/l1_gas_price_oracle.go
Comment (amit-momin) [inline core/chains/evm/gas/rollups/l1_gas_price_oracle.go:76]: Actually found one thing. For the gas estimator to initialize the L1 oracle for Scroll, we may need to add it to ths supportedChainTypes list below. I couldn't add the comment to the actual line. This list would be used [here](https://github.com/smartcontractkit/chainlink/blob/c2115d32801930fe9448b8b0ed036c2356c94776/core/chains/evm/gas/rollups/l1_gas_price_oracle.go#L72).

Comment (stackman27) [reply core/chains/evm/gas/rollups/l1_gas_price_oracle.go:76]: added! thank you for catching this

**Review by amit-momin — APPROVED** [review]

**Review by prashantkumar1982 — APPROVED** [review]

**Pull request merged**
